internal/cmd/io.go

```diff
@@ -36,3 +39,17 @@ type UpstreamIO interface {
 	// return `""`.
 	Terminal() bool
 }
+
+func NewUpstreamIO(ctx context.Context, id string, stdout string, stderr string, stdin string, terminal bool) (UpstreamIO, error) {
+	u, err := url.Parse(stdout)
+
+	if err != nil || u.Scheme == "" {
+		return NewNpipeIO(ctx, stdin, stdout, stderr, terminal)
+	}
+
+	if u.Scheme != "binary" {
```

Comment:
Maybe add a comment here that we only support named pipe and binary logger IO for containers

Reply:
I'll add docs